test/controllers/declaration/BorderTransportControllerSpec.scala

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.declaration
+
+import base.CSRFUtil._
+import base.CustomExportsBaseSpec
+import forms.Choice
+import forms.Choice.choiceId
+import forms.declaration.CommodityMeasure.commodityFormId
+import forms.declaration.{BorderTransport, CommodityMeasure}
+import generators.Generators
+import org.mockito.ArgumentMatchers
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{reset, verify}
+import org.scalacheck.Arbitrary._
+import org.scalatest.prop.PropertyChecks
+import play.api.data.Form
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import play.twirl.api.Html
+import uk.gov.hmrc.auth.core.InsufficientEnrolments
+import views.html.declaration.border_transport
+
+class BorderTransportControllerSpec extends CustomExportsBaseSpec with Generators with PropertyChecks {
+
+  private val uri = uriWithContextPath("/declaration/border-transport")
+
+  val form: Form[BorderTransport] = Form(BorderTransport.formMapping)
+
+  def view(form: Form[BorderTransport] = form, request: FakeRequest[_]): Html =
+    border_transport(form)(request, messages, appConfig)
+
+  before {
+    authorizedUser()
+    withCaching[BorderTransport](None, BorderTransport.formId)
+    withCaching[Choice](Some(Choice(Choice.AllowedChoiceValues.SupplementaryDec)), choiceId)
```

Comment:
Missing tests for standard dec? or this page is the same for both?

Reply:
this page is same for both